ocs_ci/ocs/constants.py

```diff
@@ -2067,3 +2067,54 @@
 
 
 WARP_CLIENT_PORT = 7761
+
+UI_INPUT_RULES_GENERAL = {
+    "rule1": "Starts and ends with a lowercase letter or number",
+    "rule2": "Only lowercase letters, numbers, non-consecutive periods, or hyphens",
+    "rule3": "A unique name for the BackingStore within the project",
+    "rule4": "Cannot be used before",
+    "rule5": "No more than 253 characters",
+}
+
+UI_INPUT_RULES_BACKING_STORE = {
+    "rule1": "No more than 43 characters",
+    "rule2": UI_INPUT_RULES_GENERAL["rule1"],
+    "rule3": UI_INPUT_RULES_GENERAL["rule2"],
+    "rule4": "A unique name for the BackingStore within the project",
```

Comment:
```suggestion
    "rule4": UI_INPUT_RULES_GENERAL["rule3"],
```
Can this be done?

Reply:
I am removing "rule4": "A unique name for the BackingStore within the project from UI_INPUT_RULES_GENERAL, it is actually unique to UI_INPUT_RULES_BACKING_STORE